lib/phoenix.ex

```diff
@@ -44,7 +44,9 @@ defmodule Phoenix do
       :erlang.system_flag(:backtrace_depth, stacktrace_depth)
     end
 
-    Phoenix.Logger.install()
+    if Application.get_env(:phoenix, :logger, true) do
```

Comment:
The default value should be set in the `mix.exs` file and we should use `fetch_env!` here.